src/components/Tokens/TokenDetails/TransactionsTable.tsx

```diff
@@ -0,0 +1,118 @@
+import { Table } from 'components/Table'
+import { LinkCell, TextCell } from 'components/Table/Cells'
+import { useActiveLocale } from 'hooks/useActiveLocale'
+import { useMemo } from 'react'
+import { Column } from 'react-table'
+import { useTheme } from 'styled-components'
+import { shortenAddress } from 'utils/addresses'
+import { NumberType, useFormatter } from 'utils/formatNumbers'
+
+import { mockSwapData } from './mockData'
+import { Swap, SwapAction, SwapInOut } from './types'
+import { getLocaleTimeString, getSwapType } from './utils'
+
+enum ColumnHeader {
+  Time = 'Time',
+  Type = 'Type',
+  Amount = 'Amount',
+  For = 'For',
+  USD = 'USD',
+  Maker = 'Maker',
+}
+
+export function TransactionsTable({ referenceTokenAddress }: { referenceTokenAddress: string }) {
```

Comment:
The TX Table's colors and columns aren't matching what I'm seeing in the [Figma](https://www.figma.com/file/2yKuZXmQ2UvgBbXR3txz4O/Dot-Info?type=design&node-id=684%3A1452112&mode=dev) should I be looking somewhere else?
![Screenshot 2023-09-25 at 1 48 02 PM](https://github.com/Uniswap/interface/assets/11512321/464062ad-16a0-4cf8-bf0f-5e9c934693f0)

Reply:
I've removed the coloring based on the latest designs. TDP will be a neutral view in order to reduce the cognitive load of exploring all this data.